netam/dasm.py

```diff
@@ -15,7 +15,6 @@
 
 
 class DASMDataset(DXSMDataset):
-    prefix = "dasm"
```

Comment:
@willdumm I have no idea what this prefix was about, and it doesn't seem to be used. If I'm missing something please let me know!

Reply:
Oh, right, we're just using it on the burrito. Not sure why it's also on the dataset class, I agree that can be removed.

Looks like the (burrito class) prefix variable is used in these places:
```
dnsm-experiments-1/dnsmex/dxsm_zoo.py|128 col 21| if burrito_class.prefix == "dasm":
dnsm-experiments-1/dnsmex/dxsm_zoo.py|140 col 25| if burrito_class.prefix != model_name[: len(burrito_class.prefix)]:
dnsm-experiments-1/dnsmex/dxsm_zoo.py|140 col 66| if burrito_class.prefix != model_name[: len(burrito_class.prefix)]:
dnsm-experiments-1/dnsmex/dxsm_zoo.py|405 col 22| + burrito_cls.prefix
```

Reply:
... which I see you've changed on the other PR 👍